File: app/src/main/java/com/temperoni/recipes/ui/views/StepEntry.java
package com.temperoni.recipes.ui.views;

import android.content.Context;
import android.support.v7.widget.AppCompatTextView;

import com.temperoni.recipes.R;

/**
 * @author Leandro Temperoni
 */
public class StepEntry extends AppCompatTextView {

    public StepEntry(Context context) {
        super(context);
        inflate(getContext(), R.layout.step_entry, null);
    }
}

File: app/src/main/java/com/temperoni/recipes/ui/views/StepEntry.kt
package com.temperoni.recipes.ui.views

import android.content.Context
import android.support.v7.widget.AppCompatTextView
import android.view.View

import com.temperoni.recipes.R

/**
 * @author Leandro Temperoni
 */
class StepEntry(context: Context) : AppCompatTextView(context) {

    init {
        View.inflate(getContext(), R.layout.step_entry, null)
    }
}
